ra/ra.go

```diff
@@ -676,6 +697,14 @@ func (ra *RegistrationAuthorityImpl) checkAuthorizations(ctx context.Context, na
 			return berrors.InternalServerError("found an authorization with a nil Expires field: id %s", authz.ID)
 		} else if authz.Expires.Before(now) {
 			badNames = append(badNames, name)
+		} else if authz.Expires.Before(caaRecheckTime) {
+			recheckNames = append(recheckNames, name)
```

Comment:
Do you think we might find value in having a counter for how many times we recheck CAA because of the authz being too old?